ui/components/multichain/pages/page/components/header/index.scss

```diff
@@ -0,0 +1,33 @@
+.multichain-page-header {
+  $height-screen-sm-max: 100%;
+  $width-screen-sm-min: 85vw;
+  $width-screen-md-min: 80vw;
+  $width-screen-lg-min: 62vw;
```

Comment:
I feel like these should be stored in a global stylesheet somewhere as sass variables they seem to be used a lot?